port/protobuf.h

```diff
@@ -87,12 +87,14 @@ inline bool HasPresence(const google::protobuf::FieldDescriptor& descriptor) {
 #endif
 }
 
+constexpr bool TextParserCanSetRecursionLimit();
+constexpr bool TextParserCanAllowUnknownField();
+
 inline void PrepareTextParser(google::protobuf::TextFormat::Parser& parser) {
```

Comment:
why not just reorder them?

Reply:
@vitalybuka onyl for smaller diff, as I said, I can offer flipping order. Give me a second…

Reply:
@vitalybuka pushed

Reply:
@vitalybuka I notice now that this part of the fix won't work at all. I'll need to drop the second commit. Pushed.